### DISCUSSION TITLE
Third-Party -> googletest changes

### DISCUSSION THREAD

**Discussion by GaryElshaw** [transmission/transmission#4146] [General]
I've been keeping 1/8th of an eye on the changes to googletest since the move away from releases, and in the last couple of weeks this has landed and thought it might be useful to make a mental note  for contributors and maintainers.

@coeur @ckerr @mikedld 

https://github.com/google/googletest/commit/90171d106135fc03db211fcfe928527f3cb74a5e

**Comment by Coeur** [comment]
We're already using commits for all our third-parties:
https://github.com/transmission/transmission/tree/main/third-party

**Reply by GaryElshaw** [reply]
Sure thing, i just thought it was worth mentioning because of their decisiveness in *not* having numbered releases.
